index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./styles.css" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <header id="header">
+        <nav>
+            <ul id="navbar-ul">
+                <li><button>Menu</button></li>
+                <li>
+                    <a href="https://www.youtube.com">Home</a>
+                </li>
+                <li>
+                    <input type="search">
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <section id="main-vid">
+        <iframe width="100%" height="500" src="https://www.youtube.com/embed/Yw74sDWPH7U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    </section>
+    
+    <div id="desc-and-preview">
+        <div id="description">
+            <h2>Steve Vai - "Tender Surrender"</h2>
+            <h3>Views: 1554685</h3>
+            <h4>1 mar 2008</h4>
+            <p>Steve Vai performs "Tender Surrender" from the DVD "AlienLove Secrets" featuring full-length performance videos of everysong from the "Alien Love Secrets" EP.</p>
+            <pre>
+                The DVD includes:
+                
+                • 7 Performance Videos
+                • 5.1 Surround & Stereo Mixes
+                • Alternate Video Angles
+                • Feature-length Commentary
+                • Interview with Steve Vai
+            </pre>
+        </div>
```

Comment:
Hi, @Mar12358 Please make sure that your comments are polite, as suggestions instead of orders. :pray: 

For example: :point_down: 

* This is a polite suggestion: *"I think it would be better to use semantic tags"*
* This reads as an order: *"Use semantic tags"*